src/app/components/CallToActionLinkComponent/index.tsx

```diff
@@ -0,0 +1,48 @@
+/** @jsx jsx */
+import { PropsWithChildren } from 'react';
+import { jsx } from '@emotion/react';
+import useClickTrackerHandler from '#app/hooks/useClickTrackerHandler';
+import { EventTrackingMetadata } from '#app/models/types/eventTracking';
+import Chevron from './Chevron';
+import FlexWrapper from './FlexWrapper';
+import Text from './Text';
+import styles from './index.styles';
+
+type CallToActionLinkProps = {
+  to?: string;
```

Comment:
```suggestion
  url: string;
```
I can understand why this was named as such ('CallToActionLink' _to_ some URL I guess?), but I think it would be clearer if this was something more explicit like `url`. Also, shouldn't this be required if the main purpose of the component is to render a link?

Reply:
Good point!